lighthouse-cli/bin.js

```diff
@@ -48,16 +48,16 @@ if (cliFlags.listTraceCategories) {
 /** @type {string} */
 const url = cliFlags._[0];
 
-/** @type {!LH.Config|undefined} */
+/** @type {LH.Config.Json|undefined} */
 let config;
 if (cliFlags.configPath) {
   // Resolve the config file path relative to where cli was called.
   cliFlags.configPath = path.resolve(process.cwd(), cliFlags.configPath);
-  config = /** @type {!LH.Config} */ (require(cliFlags.configPath));
+  config = /** @type {LH.Config.Json} */ (require(cliFlags.configPath));
 } else if (cliFlags.perf) {
-  config = /** @type {!LH.Config} */ (perfOnlyConfig);
+  config = perfOnlyConfig;
```

Comment:
since these are now `js` files, typescript now imports them and checks that they're compatible with being assigned to `LH.Config.Json|undefined` here :)